src/tonic/mod.rs

```diff
@@ -46,6 +49,31 @@ impl Qdrant for QdrantService {
     }
 }
 
+pub struct QdrantInternalService {
+    /// HTTP port accessible from inside the cluster
+    http_port: u16,
```

Comment:
Do we want this to be configurable? It now uses `service.http_port`.

Should a user be able to configure a custom HTTP port for internal communication? Or should the user be able to disable this HTTP port for internal use all together? This may be useful for specialized cluster configurations where the HTTP is not forwarded, or bound to a different port.

Reply:
I don't think we need any other port for now